### PR TITLE
Connecting to the info page

### PR DESCRIPTION
**Pull request by mwernimont**
@isuftin Look correct?

### CODE REVIEW

File: coastal-hazards-portal/src/main/webapp/js/cch/objects/front/Intro.js [hunk 1]
```diff
@@ -129,6 +129,11 @@ CCH.intro = (function () {
 
 				}
 			},
+                        {
+				onbeforechange:function(){
+                                    window.location.href = CCH.CONFIG.contextPath = '../info/#mapContentArea';
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/front/Intro.js]: What youre doing here is forcibly redirecting the browser to another url. This is not what we want. We want to give the user the option (via a button) to go back to the info page

File: coastal-hazards-portal/src/main/webapp/js/cch/objects/front/Intro.js [hunk 2]
```diff
@@ -287,6 +292,11 @@ CCH.intro = (function () {
 					}
 				}
 
+			},
+                        {
+				onbeforechange:function(){
+                                    window.location.href = CCH.CONFIG.contextPath = '../../info/#bucketContentArea';
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/front/Intro.js]: Same as above

**Comment by mwernimont** [comment]
Well really kinda lost then, I have no knowledge in this stuff

**Comment by isuftin** [comment]
?

**Comment by mwernimont** [comment]
When I was given the task it was simple add the steps to the tutorial, now its programming it to do things at certain times and complexity above my head.  Also seems inefficient to have someone with little JS background to be doing this sort of thing when a more experienced programmer could do it in minutes over the hours I've put into it. 

All I'm saying it went from simple to complex and we still have the novice on it

**Comment by isuftin** [comment]
All this requires is a button or text link to be put onto the identification/navigation bubble at specific steps that allows a user to navigate back to the info page

**Comment by mwernimont** [comment]
Fixed Firefox too, so just mobile to fix as of yet, and maybe IE

**Comment by isuftin** [comment]
Sweet!

**Comment by mwernimont** [comment]
Mergable?

**Comment by isuftin** [comment]
Sure let's do that

**Pull request merged**